runtime/drivers/sqlite/registry.go

```diff
@@ -91,3 +117,13 @@ func (c *connection) DeleteInstance(_ context.Context, id string) error {
 	_, err := c.db.ExecContext(ctx, "DELETE FROM instances WHERE id=$1", id)
 	return err
 }
+
+func toJSON(data map[string]string) ([]byte, error) {
+	return json.Marshal(data)
+}
+
+func fromJSON(data []byte) (map[string]string, error) {
```

Comment:
Maybe add prefix to clarify context, e.g. `envToJSON` or `mapToJSON` (and some for `fromJSON`)

Reply:
I again got confused with Go's naming conventions. Thought map is implicit from datatype of arg/returned object.